src/components/ReportManagement.vue

```diff
@@ -84,11 +117,12 @@ function setElementBlur() {
         <span class="icon-center">Hilfskraft-Verträge verwalten:</span>
         <HelpDialog></HelpDialog>
       </div>
-      <div class="d-flex align-center">
+      <div class="d-flex align-center mb-3">
         <v-text-field
           v-model="newReference"
           class="mr-3"
           label="Neuen Hilfskraft-Vertrag zuordnen"
+          :rules="rules"
           @keydown.enter="addReference"
         >
           <template #append-innner>
```

Comment:
The "add"7"Hinzufügen" button should also be disabled if an entry is not according to UUIDv4 format.